packages/flutter/lib/src/material/switch_list_tile.dart

```diff
@@ -284,10 +284,15 @@ class SwitchListTile extends StatelessWidget {
        assert(autofocus != null),
        super(key: key);
 
-  /// Creates the wrapped switch with [Switch.adaptive].
+  /// Creates a Material [ListTile] with an adaptive [Switch], following
+  /// Material design's
+  /// [Cross-platform guidelines](https://material.io/design/platform-guidance/cross-platform-adaptation.html).
   ///
-  /// Creates a [CupertinoSwitch] if the target platform is iOS, creates a
-  /// material design switch otherwise.
+  /// {@macro flutter.material.switch.adaptive}
+  ///
+  /// This constructor creates a Material [ListTile] with a platform-dependent
```

Comment:
I think this overstates the truth. We're only "adapting" for iOS, i.e. we're adapting the ListTile's Material switch component to look a little more familiar when it appears on iOS.